elixir/apps/api/test/support/channel_case.ex

```diff
@@ -14,6 +14,7 @@ defmodule API.ChannelCase do
       import Phoenix.ChannelTest
       import API.ChannelCase
       alias Domain.Repo
+      alias Domain.Fixtures
```

Comment:
@bmanifold I streamlined fixture naming so there is no need to `alias` a ton of fixtures/factories in our tests, we can just use `Fixtures.NameOfDomainContext` anywhere.

Reply:
👍